### DISCUSSION TITLE
Is there any feature development planned for Newrelic workflows ?

### DISCUSSION THREAD

**Discussion by mayureshbhardwaj** [newrelic/terraform-provider-newrelic#1824] [Q&A] [answered]
I recently completed a project for managing our new relic configs, alerting using terraform. I noticed that for notifications there is an EOL set and Newrelic asks to use Workflows. 

Is there any plan to add New relic Workflow support in terraform provider?

**Comment by kidk** [comment, marked as answer]
Thanks for question @mbhardwaj1990 

I've added a reply here #1790 It's best to track the work there.
